include/nbl/video/surface/ISurface.h

```diff
@@ -8,6 +8,8 @@
 
 #include "nbl/video/IAPIConnection.h"
 
+#include "nbl/builtin/glsl/utils/swapchain_transform_e.h"
```

Comment:
include them closer to the `static_assert` and `#undef` right after to prevent preprocessor pollution